riscv-plic.adoc

```diff
@@ -176,16 +176,17 @@ Each global interrupt can be enabled by setting the corresponding bit in the
 `enables` register. The `enables` registers are accessed as a contiguous array
 of 32-bit registers, packed the same way as the `pending` bits. Bit 0 of enable
 register 0 represents the non-existent interrupt ID 0 and is hardwired to 0.
-PLIC has 15872 Interrupt Enable blocks for the contexts. The `context` is referred 
+PLIC has 15872 Interrupt Enable blocks for the contexts. 
+
+The `context` is referred 
 to the specific privilege mode in the specific Hart of specific RISC-V processor 
-instance. How PLIC organizes interrupts for the contexts (Hart and privilege mode) 
+instance. For example, in an 4-core system with 2-way SMT, you have 8 harts and probably
+at least two privilege modes per hart: machine mode and supervisor mode.
+(ref: https://github.com/torvalds/linux/blob/5bfc75d92efd494db37f5c4c173d3639d4772966/Documentation/devicetree/bindings/interrupt-controller/sifive%2Cplic-1.0.0.yaml)
+
+How PLIC organizes interrupts for the contexts (Hart and privilege mode) 
```

Comment:
This is my typo when copy and paste. should be
"However PLIC organizes ..."

Reply:
I had fixed the typo, but why it should be `However` instead of `How` ? The original one before this pr is `How`.

Reply:
Oops... my bad. I thought it was a typo. Please help me to change it back to "How".

Reply:
Ok, I had fixed it.